Sources/DesignLibrary/ArtemisWebView.swift

```diff
@@ -12,16 +12,31 @@ public struct ArtemisWebView: UIViewRepresentable {
 
     @Binding var urlRequest: URLRequest
     @Binding var contentHeight: CGFloat
+    @Binding var isLoading: Bool
```

Comment:
Why not use a @ State instead? 🤔 I assume this binding is used for conditionally rendering a ProgressView on a parent view but I guess this could be done directly inside ArtemisWebView.

Reply:
Yes it's for rendering a ProgressView or something else. I thought this way it's more customisable by the parent view. E.g. maybe you don't want to show the webview at all while loading. But if you think this will not be necessary I can change it

Reply:
Oh, I see. Let's keep it customizable then 👍